src/components/ContactForm/ContactForm.js

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components';
+import Input from '../Input/Input';
+import emailjs from 'emailjs-com';
+
+export default function ContactForm() {
+  return (
+    <StyledContactForm aria-labelledby="formHeader" onSubmit={onSubmit}>
+      <h2 id="formHeader">Contact Us</h2>
+      <Input
+        labelText="Restaurantname"
+        id="restaurantname"
+        name="restaurantname"
+        type="text"
+        required={true}
```

Comment:
I'm not sure, but I think you could set the required without `true`. Try it out for your input fields
```js
required
```